test/test_framework.py

```diff
@@ -84,7 +89,13 @@ class TestFramework:
     def test_deprecated_init(self, caplog: pytest.LogCaptureFixture):
         # For 0.7, this still works, but it is deprecated.
         with caplog.at_level(logging.WARNING):
-            framework = ops.Framework(':memory:', None, None, None)  # type: ignore
+            framework = ops.Framework(
+                ':memory:',  # type: ignore
+                None,  # type: ignore
+                None,  # type: ignore
+                None,  # type: ignore
+                juju_debug_at=_JujuContext.from_dict({'JUJU_VERSION': '0.0.0'}).debug_at,
```

Comment:
This seems a long-winded way of saying `set()`? Same below.

No need to change it if you have some reason for preferring this way.

Reply:
I used this way to mimic how the `_Manager` in `main.py` creates the `Framework` in the `_make_framework` function; also it shows some context of where the value comes from. But after a second look, I guess it's not super necessary, I will change it to `set()` (and other occurrences, too).